pom.xml

```diff
@@ -292,7 +292,11 @@
                     <version>2.22.0</version>
                         <configuration>
                             <!-- #1186 Temporarily skip a test. -->
-                            <excludes>org.airsonic.player.service.search.*TestCase.java</excludes>
+                            <excludes>
+                                <exclude>
+                                    org.airsonic.player.service.search.*TestCase.java
```

Comment:
All the examples in the documentation are shown without additional whitespace.  Should this just be:
`<exclude>org.airsonic.player.service.search.*TestCase.java</exclude>`?